test/assert.js

```diff
@@ -778,7 +794,7 @@ describe('assert', function () {
       var errMap = new Map();
 
       errMap.set({1: 20}, 'number');
-
+   
```

Comment:
haha

I added a `.editorconfig` file on my home with some rules that I always want.
I think it goes looking upwards until it finds one